xmovie/presets.py

```diff
@@ -215,12 +215,12 @@ def _add_coast(ax, style):
 
 ### Presets (should proabably put all others into a submodule)
 def basic(
-    da, fig, timestamp, plotmethod=None, plot_variable=None, subplot_kw=None, **kwargs
+    da, fig, timestamp, framedim="time", plotmethod=None, plot_variable=None, subplot_kw=None, **kwargs
```

Comment:
I like this way of doing it. The public presets have framedim as kwarg and the private ones as an explicit arg! Nice.